src/main/java/org/sonar/plugins/coverage/generic/ReportParser.java

```diff
@@ -45,12 +45,13 @@
 public class ReportParser {
 
   public enum Mode {
-    COVERAGE, IT_COVERAGE, UNITTEST
+    COVERAGE, IT_COVERAGE, OVERALL_COVERAGE, UNITTEST
   }
 
   private static final Map<Mode, String> ROOT_NODE = ImmutableMap.<Mode, String>builder()
     .put(Mode.COVERAGE, "coverage")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "coverage" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)